api/google_test.go

```diff
@@ -77,5 +77,19 @@ func TestGetGoogleSCCHealthStatus(t *testing.T) {
 		status := GetSCCHealthStatus(x.serverURL)
 		assert.Equal(t, x.status, status, "Test case %d status check failed", i)
 	}
+}
+
+func TestGetSCCLatestEventTime(t *testing.T) {
+	x, err := GetSCCLatestEventTime(nil)
+	assert.Nil(t, x)
+	assert.Error(t, err, "no authentication present should result in error")
+}
 
+func TestGetSCCSourcesByName(t *testing.T) {
+	x, err := GetSCCSourcesByName("", "bad_regex(")
```

Comment:
This test also exercises only 1 specific case and it's name doesn't reflect that. Consider defining a range of test cases covering corner cases in a table structure and testing them sequentially within a single test func.